content/uiux/concepts/ux-psychology/terms/jacobs-law/jacobs-law.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: "Jacob's Law"
```

Comment:
Please avoid using `""` instead of `''` to keep the content consistent with other doc entries.

```suggestion
Title: 'Jacobs Law'
```